android/src/main/java/com/dooboolab.audiorecorderplayer/RNAudioRecorderPlayerModule.kt

```diff
@@ -172,7 +192,6 @@ class RNAudioRecorderPlayerModule(private val reactContext: ReactApplicationCont
 
         try {
             mediaRecorder!!.stop()
-            mediaRecorder!!.reset()
```

Comment:
@jbaudanza Could you kindly check why `reset` should be removed?

Reply:
I'll leave this part as remaining question and will merge this `PR`! Thanks @jbaudanza